src/components/ModalManager/Modal.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import styled from 'styled-components'
+import EscapeOutside from '../EscapeOutside/EscapeOutside'
+import { Button, Text, breakpoint } from '@aragon/ui'
+import { noop } from '../../utils'
+
+const Modal = ({ title, body, moreText, onHide, More, blocking }) => (
+  <Main>
+    <WrapModal role="alertdialog" onEscapeOutside={blocking ? noop : onHide}>
+      <Header>
+        <Text size="xxlarge">{title}</Text>
+      </Header>
+      <Body>{body}</Body>
+      <Footer>
+        <Button mode="text" onClick={onHide}>
+          Close
+        </Button>
+        {More}
+      </Footer>
+    </WrapModal>
+  </Main>
+)
+
+const Main = styled.div`
+  display: grid;
+  grid-template-rows: auto 1fr auto;
+  padding: 20px;
+`
+
+const WrapModal = styled(EscapeOutside)`
+  background: #fff;
+  padding: 40px 40px 20px;
+  box-shadow: 0 10px 28px rgba(0, 0, 0, 0.2);
+  border-radius: 4px;
+
+  /* desktop */
+  ${breakpoint(
+    'medium',
+    `
+      height: unset;
+      min-height: 40vh;
+      padding: 3.5em;
+      margin: auto;
+    `
+  )};
+
+  ${breakpoint('large', `max-width: 70%;`)};
```

Comment:
What do you think about adding a `max-width: 800px` to all breakpoints? On larger displays this can look quite awkward:

<img width="1171" alt="screen shot 2018-10-12 at 11 08 58 am" src="https://user-images.githubusercontent.com/4166642/46859880-4eff8500-ce0f-11e8-8b0a-36ddfeccc8cd.png">